wcc-contentful-app/lib/wcc/contentful/app.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'wcc/contentful/rails'
 
+require_relative './app/rails'
```

Comment:
after testing this branch in reengage with Rails 7.1 I found this is needed so the "engine" features work (access to helpers, views, etc, without this it fails to find MenuHelper, SectionHelper, and the views from wcc-contentful-app).

seems like Rails 7.0 was loading those modules implicitly but we have to be explicit now